app/swagger/requests/gibct/institution_programs.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
```

Comment:
Please add examples to your swagger docs.  This would be a lot more clear if realistic examples were used instead of `"string`
<img width="1046" alt="Screen Shot 2019-10-21 at 5 24 12 PM" src="https://user-images.githubusercontent.com/19188/67244593-0ae28f80-f428-11e9-8d0d-5b7f45fd1caa.png">